frontend/src/forms/ComponentLike/Probes.tsx

```diff
@@ -128,14 +127,12 @@ class RenderProbe extends React.PureComponent<Props> {
               component={RenderNestedTextfield}
               placeholder="8080"
               parse={NormalizePositiveNumber}
-              validate={ValidatorRequired}
               style={{ width: 60 }}
             />
             <Field
               name={`${name}.httpGet.path`}
               component={RenderNestedTextfield}
               placeholder="/healthy"
```

Comment:
This field need a parse attribute. If it's not blank, it should have prefix "/".